src/config/Network.js

```diff
@@ -0,0 +1,5 @@
+export const networks = {
+  ROPSTEN: "ROPSTEN",
```

Comment:
Do you need these constants ?

Reply:
https://github.com/singnet/refactored-dapp/pull/184/commits/e82e1f8abc060c8f9402b50f18a23d34e0386ee3